crates/bevy_asset/src/assets.rs

```diff
@@ -305,3 +311,19 @@ impl AddAsset for App {
         self
     }
 }
+
+#[test]
```

Comment:
IMO we should always be keeping tests in #[cfg(test)] bound submodules to avoid compiling them during non-test use.